config.py

```
@@ -60,14 +63,101 @@ def _detect_system_memory():
+}
+
 # --- Confidence thresholds ---------------------------------------------------
-TAU_FIELD_ACCEPT  = 0.95  # auto-accept threshold
-TAU_ENHANCER_PASS = 0.90  # after enhancer / alt engine
-TAU_LLM_PASS      = 0.85  # LLM fallback
+# These control the OCR pipeline's decision points:
+# - TAU_FIELD_ACCEPT: High confidence threshold for immediate acceptance
+# - TAU_ENHANCER_PASS: Medium confidence threshold for enhanced processing
+# - TAU_LLM_PASS: Low confidence threshold for LLM fallback
+
+import os
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)